src/streamlit_extras/dataframe_explorer/__init__.py

```diff
@@ -178,7 +179,7 @@ def generate_fake_dataframe(size, cols, col_names=None, intervals=None, seed=Non
     default_intervals = {
         "i": (0, 10),
         "f": (0, 100),
-        "c": ("names", 5),
+        "c": ("names", 12),
```

Comment:
Made this in order to have enough unique values to unlock a `text_input()` and not a `selectbox()`